src/services/polkadot/index.tsx

```diff
@@ -202,7 +202,7 @@ function checkBalancePeriodically(
       try {
         const someBalanceUnits = await getStellarBalanceUnits(
           stellarTargetAccountId,
-          outputToken.stellarAsset.code.stringRaw,
```

Comment:
This one was wrong to have been `raw` in the first place, as it compares with the return of the stellar balance query.